morph_tool/__init__.py

```diff
@@ -17,7 +17,3 @@ class NoAxonException(MorphToolException):
 
 class NoDendriteException(MorphToolException):
     '''MorphTool exception'''
-
-
-class TooManyAxonsException(MorphToolException):
-    '''MorphTool exception'''
```

Comment:
Thanks for the comment @wizmer. Tests in morphology-repair-workflow fails because `morphology_repair_workflow/graft_axons.py` explicitly catches all the exceptions
`NoAxonException, TooManyAxonsException, NoDendriteException, MorphToolException`.
Anyway, considering that NoAxonException, TooManyAxonsException, NoDendriteException are subclasses of MorphToolException, the code in morphology-repair-workflow could be changed to catch only MorphToolException.
Alternatively, for backward compatibility, I could (temporarily?) keep TooManyAxonsException here, even though it's not used anymore in morph-tool.
Do you know any other library that could rely explicitly on this exception?
This could be also mentioned in a changelog file, if it makes sense.

Reply:
I think we can merge this and fix the failing tests in morphology-repair-workflow. Could you take care of it ?

Reply:
Sure 👍